src/SubmitPage/SubmitForm.js

```diff
@@ -27,6 +32,8 @@ const {
   PLACE_ID,
 } = FORM_FIELDS;
 
+const enableTags = true;
```

Comment:
this is a feature flag

Reply:
once all changes are reviewed, i'll set it to false before merging. since backend is not persisting these new fields yet

Reply:
thanks for adding this!